packages/mobile/src/components/usdc-manual-transfer-drawer/USDCManualTransferDrawer.tsx

```diff
@@ -97,10 +109,24 @@ const useStyles = makeStyles(({ spacing, palette, typography }) => ({
 
 export const USDCManualTransferDrawer = () => {
   const styles = useStyles()
+  const dispatch = useDispatch()
   const { toast } = useToast()
-  const { isOpen, onClose, onClosed } = useUSDCManualTransferModal()
+  const { isOpen, onClose, onClosed, data } = useUSDCManualTransferModal()
+  const { source, amount: amountInCents, startPurchaseParams } = data ?? {}
   const { onPress: onPressLearnMore } = useLink(USDCLearnMore)
-  const neutral = useColor('neutral')
+  const { neutral, specialLightGreen } = useThemeColors()
+
+  const stage = useSelector(getPurchaseContentFlowStage)
+  const error = useSelector(getPurchaseContentError)
+  const isUnlocking = !error && isContentPurchaseInProgress(stage)
+  const { data: balanceBN } = useUSDCBalance({
+    isPolling: true,
+    pollingInterval: 1000
+  })
+  const balance = USDC((balanceBN ?? new BN(0)) as BN).value
```

Comment:
Do you still get errors without `as BN` here? You shouldn't... it should take a `BNUSDC` fine?